lib/datasource/remote/api/tokenmodels_repository.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:async/async.dart';
+import 'package:http/http.dart' as http;
+import 'package:seeds/datasource/remote/api/http_repo/http_repository.dart';
+import 'package:seeds/datasource/remote/api/http_repo/seeds_scopes.dart';
+import 'package:seeds/datasource/remote/api/http_repo/seeds_tables.dart';
+
+/// Retrieve token metadata (used for display of currency cards etc) from
+/// a table in the token master smart contract (tmastr.seeds)
+class TokenModelsRepository extends HttpRepository {
+  Future<Result<List<int>>> getAcceptedTokenIds(String useCase) async {
+    final request = createRequest(
+      code: SeedsCode.accountTokenModels,
+      scope: useCase,
+      table: SeedsTable.tableTokenMasterAcceptances,
+      limit: 100,
+    );
+    return http
+        .post(Uri.parse('$baseURL/v1/chain/get_table_rows'),
+            headers: headers, body: request)
+        .then((http.Response response) =>
+            mapHttpResponse<List<int>>(response, (dynamic body) {
+              final acceptances = body['rows'].toList();
+              final tokenIds = List<int>.from(
+                  acceptances.map((row) => row['token_id']).toList());
+              return tokenIds;
+            }))
+        .catchError((error) => mapHttpError(error));
+  }
+
+  Future<Result<Map<String, dynamic>>> getMasterTokenTable() async {
```

Comment:
what is the master token table?

Can it have more than 100 entries? Does it have more than 10?